src/python/WMCore/BossAir/Plugins/SimpleCondorPlugin.py

```diff
@@ -283,8 +284,11 @@ def complete(self, jobs):
                         condorFilePath = os.path.join(job['cache_dir'], condorFile)
                         if os.path.isfile(condorFilePath):
                             logTail = BasicAlgos.tail(condorFilePath, 50)
-                            logOutput += 'Adding end of %s to error message:\n' % condorFile
-                            logOutput += '\n'.join(logTail)
+                            logOutput += 'Adding end of %s to error message:\n\n' % condorFile
```

Comment:
It will have the 
<@========== WMException Start ==========@>
That is why I removed it.